pubsub/rabbitmq/rabbitmq.go

```diff
@@ -432,6 +434,18 @@ func (r *rabbitMQ) prepareSubscription(channel rabbitMQChannelBroker, req pubsub
 		args[amqp.QueueTypeArg] = amqp.QueueTypeClassic
 	}
 
+	// Applying x-single-active-consumer if defined at subscription level
+	if val := req.Metadata[reqMetadataSingleActiveConsumerKey]; val != "" {
+		parsedVal, pErr := strconv.ParseBool(val)
```

Comment:
Do not use `strconv.ParseBool` instead use our preferred method from package `dapr/kit/utils` called `IsTruthy` https://github.com/dapr/kit/blob/106329e5839f70f3234b9f49b7dabff0247c0a90/utils/strings.go#L23

That will allow a lot more values to be used to express the "true" / on state.

Reply:
Thanks for highlighting this, the code is far more readable this way 👍